docs/user/config/tsConfig.md

```diff
@@ -2,9 +2,9 @@
 title: TypeScript Config option
 ---
 
-The `tsConfig` option allows you to define the which `tsconfig` JSON file to use. An inline compiler options object can also be specified instead of the path to a file.
+The `tsConfig` option (alias `tsconfig`) allows you to define which `tsconfig` JSON file to use. An inline [compiler options][] object can also be specified instead of a file path.
```

Comment:
what do you mean with `[compiler options][]` ? The empty square bracket looks strange.

Reply:
That’s Markdown syntax for a link. Reference is at the bottom of the page.

Reply:
I see 👍